pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -439,17 +459,45 @@ func registerClusterToCluster(r registry.Registry) {
 					return lv.pollLatency(ctx, setup.dst.db, ingestionJobID, time.Second, workloadDoneCh)
 				})
 
-				t.Status("waiting for replication stream to finish ingesting initial scan")
+				t.L().Printf("waiting for replication stream to finish ingesting initial scan")
 				waitForHighWatermark(t, setup.dst.db, ingestionJobID, sp.timeout/2)
 
-				t.Status("waiting for src cluster workload to complete")
-				m.Wait()
+				t.Status(fmt.Sprintf(`initial scan complete. run workload and repl. stream for another %d minutes`,
+					int(sp.additionalDuration.Minutes())))
+
+				var currentTime time.Time
+				setup.dst.sql.QueryRow(t, "SELECT clock_timestamp()").Scan(&currentTime)
+				cutoverTime := currentTime.Add(sp.additionalDuration - sp.cutover)
+				t.Status("cutover time chosen: %s", cutoverTime.String())
```

Comment:
`t.Status` behaves more like `fmt.Print` than `fmt.Printf`, so you want something like:

```go
t.Status("cutover time chosen: ", cutoverTime.String())
```

This API is confusing! (recent example: #95025).

Reply:
Done.